client/src/components/CardWrapper/CardWrapper.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Box, Paper } from '@mui/material';
+
+interface CardWrapperProps {
+  children: React.ReactNode;
+}
+
+const CardWrapper: React.FC<CardWrapperProps> = ({ children }) => {
```

Comment:
I don't think we need to componentize this, since it's just a Box/Paper component with a few styles :)